### PR TITLE
Generic backup

### PR DESCRIPTION
**Pull request by Yostra**

### CODE REVIEW

File: electron-react/src/pages/ImportHexKey.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  },
+  navigator: {
+    color: "#ffffff",
+    marginTop: theme.spacing(4),
+    marginLeft: theme.spacing(4),
+    fontSize: 35
+  }
+}));
+
+const ImportHexKey = () => {
```
Comment (mariano54) [inline electron-react/src/pages/ImportHexKey.js:61]: Why is this here, we only have mnemonic keys now

File: electron-react/src/middleware/middleware_api.js [hunk 2]
```diff
@@ -166,15 +165,6 @@ export const handle_message = (store, payload) => {
     } else if (payload.origin === service_harvester) {
       store.dispatch(getPlots());
     }
-  } else if (payload.command === "log_in") {
```
Comment (mariano54) [inline electron-react/src/middleware/middleware_api.js:169]: Can you make sure to test all cases in the UI to see if there are bugs?

File: electron-react/src/pages/ImportHexKey.js (hunk 1 above)
Comment (Yostra) [reply electron-react/src/pages/ImportHexKey.js:61]: hmm, probably a rebase thing. If it got deleted on dev and I made a change there, during rebase auto-merge would ignore the delete and just apply my change. (I think)
